### PR TITLE
leaf_area_index wiki

### PR DESCRIPTION
**Pull request by LaplacetteCM**
This is the first draft of the leaf_area_index.md wiki file

### CODE REVIEW

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (morrowcj) [inline SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: To better match the code, can we do this:

```suggestion
is the first of two shape parameters (`_lai_shapes`), $l_{\text{2}}$ is the second shape parameter, and $h$ is `heat_fraction`, 

```

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (morrowcj) [inline SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: I'm not a huge fan of the "fr" notation that SWAT uses. Would it look better if we replace "fr" with a different variable? What about something like a lowercase "p" for "proportion"?

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (morrowcj) [inline SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: Similar to my above comment, I'm also not really a fan of using abbreviations for a single symbol (this could be confused for $L \times A \times I$). What if we use $A_{\text{leaf}}$ to represent leaf **area**?

I don't feel strongly about this or my previous notation comment, so, if you think it is more clear to use "LAI", feel free to say so and leave it as is.

**Review by morrowcj — APPROVED** [review]
I think this looks really good. The documentation is extremely clear and concise (less than 100 lines!). I have a few minor comments about notation, but nothing big.

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (LaplacetteCM) [reply SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: True. I think the use of LAI is very popular within the ag world and probably everyone would understand it, but in terms of the code I agree with your suggestion of using clearer notation. However, I think that we must keep the name of the variables the way it is now here and in the code (*e.g.*, Leaf area is not the same as leaf area index). 
So I would say: " $A_{\text{leaf, i}}$ is the `leaf_area_index` on a given day". Otherwise, we would be changing the meaning of the equation. Does it make sense? 

Note: I've changed both the fraction and LAI notation to A+subscripts

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (LaplacetteCM) [reply SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: perfect!

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (morrowcj) [reply SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: Oh, good point! I meant to say "leaf area index". I think your solution is great.

**Comment by morrowcj** [comment]
Looks great! I don't think we need to wait for another review, so I'm going to go ahead and force the merge, if you're ready - just let me know.

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (ew3361zh) [inline SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: It seems like it could be helpful to the reader if you have bolded a term previously, to continue bold it throughout. e.g. **canopy height** being bolded on line 10 to be consistent with its bolding on line 7.

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (ew3361zh) [inline SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: What do you think about rewording "RuFaS needs to first calculate..." to "RuFaS first calculates"? The former sounds a little like RuFaS may not currently be doing the calculating and the latter makes it sound like a description of a process that is happening. Very nitpicky so if you like it the way it's written please leave it.

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (ew3361zh) [inline SC_redesign/Crop_and_Soil/docs/leaf_area_index.md:21]: The second canopy height would not need to be bolded in my mind (despite my earlier comment) since it's not a reference to the variable.

**Review by ew3361zh — APPROVED** [review]
I think this looks great and is a helpful model for trying to create the wiki for other parts of RuFaS. I think it's helpful that you've done this at the class level. I had a couple of small comments and suggestions but otherwise really nice work and thanks for bringing me in on it!

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (LaplacetteCM) [reply SC_redesign/Crop_and_Soil/docs/leaf_area_index.md]: Totally agree, I tried to do that at the beginning of each paragraph but I should have also done that every time I used them.

File: SC_redesign/Crop_and_Soil/docs/leaf_area_index.md
Comment (LaplacetteCM) [reply SC_redesign/Crop_and_Soil/docs/leaf_area_index.md:21]: Yes, I agree!

**Pull request merged**
